### PR TITLE
Added to use templates when creating new incident

### PR DESCRIPTION
**Pull request by dem-evgeny**
Added to use templates when creating new incident

### CODE REVIEW

File: Integrations/integration-ServiceNow.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ commonfields:
   version: -1
 name: ServiceNow
 display: ServiceNow
-fromversion: 2.5.0
```
Comment (Spivaka) [inline Integrations/integration-ServiceNow.yml]: I don't think you intended to remove this.
Didn't we have SN prior to 2.5?
if so, we should add another integration with to version 2.5, before those changes

File: Integrations/integration-ServiceNow.yml [hunk 2]
```diff
@@ -96,11 +95,17 @@ script:
         }
     };
 
-    var getBody = function() {
+    var getBody = function(templateName) {
         var body = {};
+        if (templateName !== undefined) var tmpRes = SNGetTemplate(templateName);
```
Comment (Spivaka) [inline Integrations/integration-ServiceNow.yml]: don't use such if ... declaration statements, this is confusing as coders will search declarations in the left side
write it in the following manner
var tmpRes;
if (...) {
tmpRes = ...
}

File: Integrations/integration-ServiceNow.yml (hunk 1 above)
Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: fixed

File: Integrations/integration-ServiceNow.yml (hunk 2 above)
Comment (dem-evgeny) [inline Integrations/integration-ServiceNow.yml]: fixed

File: Integrations/integration-ServiceNow.yml [hunk 3]
```diff
@@ -96,11 +96,22 @@ script:
         }
     };
 
-    var getBody = function() {
+    var getBody = function(templateName) {
         var body = {};
+        var tmpRes;
+
+        if (templateName){
```
Comment (Spivaka) [inline Integrations/integration-ServiceNow.yml]: checking a !== undefined can be replaced with a, cause if a has a value if (a) return true, else false (also when a is false or empty string)

**Pull request merged**